bedrock/firefox/templates/firefox/releases/dev-browser-notes.html

```diff
@@ -1,18 +1,17 @@
 {% extends "firefox/releases/aurora-notes.html" %}
 
-{% block body_class %}fx-notes blueprint{% endblock %}
-
 {% block site_header_logo %}
-  <h2>{{ high_res_img('firefox/developer/title.png', {'alt': _('Firefox Developer Edition'), 'width': '220', 'height': '84'}) }}</h2>
+  <h2>{{ high_res_img('firefox/developer/title-inverse.png', {'alt': _('Firefox Developer Edition'), 'width': '220', 'height': '84'}) }}</h2>
 {% endblock %}
 
-{% block notes_heading_primary %}
-  <h1>{{ _('Firefox Developer Edition Notes')|f(channel=channel_name) }}</h1>
-{% endblock %}
 {% block notes_heading_secondary %}
-  <h2>{{ _('Version {version}, first offered to Firefox Developer Edition users on {date}')|f(date=release.release_date|l10n_format_date, version=release.version) }}</h2>
+  <div class="version">
+    <h3>{{ _('{version}')|f(version=release.version) }}</h3>
+    <h4>{{ _('Firefox Developer Edition') }}</h4>
+    <p>{{ _('{date}')|f(date=release.release_date|l10n_format_date) }}</p>
+  </div>
+  <div class="description">
+      <h2>{{ _('Version {version}, first offered to Firefox Developer Edition channel users on {date}')|f(channel=release.channel, date=release.release_date|l10n_format_date, version=release.version) }}</h2>
```

Comment:
Nit: Perhaps it is better to use another `h4` here as apposed to a `h2`? Looking at the semantics and display, it looks very much like it shares a lot with the `h4` above on line 10.